test/libraries/cusparse/conversions.jl

```diff
@@ -74,6 +74,24 @@ end
     end
 end
 
+if !(v"12.0" <= CUSPARSE.version() < v"12.1")
```

Comment:
Why the version guard?

Reply:
Because when NVIDIA released the version v12.0, the conversion between CSC <--> CSR was broken with 1-based indexing and small matrices.

It could work but I didn't tested.